src/jni_core/src/jvm_runtime.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use core::option::Option::Some;
+use std::ffi::c_void;
+use std::fs;
+use std::path::Path;
+use std::sync::LazyLock;
+
+use jni::strings::JNIString;
+use jni::{InitArgsBuilder, JNIVersion, JavaVM, NativeMethod};
+use risingwave_common::error::{ErrorCode, RwError};
+use risingwave_common::util::resource_util::memory::total_memory_available_bytes;
+
+use crate::run_this_func_to_get_valid_ptr_from_java_binding;
+
+pub static JVM: LazyLock<Result<JavaVM, RwError>> = LazyLock::new(|| {
+    let libs_path = if let Ok(libs_path) = std::env::var("CONNECTOR_LIBS_PATH") {
```

Comment:
I think we can define a default path such as `libs/` or `javalibs/` here, and place the `jar` files accordingly in our release packages as well as docker images. Just a random idea, can be done later.